### DISCUSSION TITLE
Support on New widget development

### DISCUSSION THREAD

**Discussion by vserefidou** [biolab/orange3#5597] [General]
Hello, 

We are looking for somebody that can develop for us a new widget for emotions analysis on text. 

Pls contact me for more info and to discuss the cost and the est. time needed. 

Many thanks in advance,
Valentina

**Comment by ajdapretnar** [comment]
Dear Valentina, all our commercial projects are done via our spin-off company [Revelo](https://revelo.ai/). You can write to info@revelo.bi for more information.

Thank you for considering Orange!

**Reply by vserefidou** [reply]
thank you so much, Ajda!

I will send the email to Revelo right now. 

Have a wonderful day,
Valentina
